### PR TITLE
aligning public apis of Time,Timer and Stopwatch

### LINKED ISSUE #15834: Make methods on `Time`, `Timer`, and `Stopwatch` more consistent
## Bevy version

0.14 / main

## What you did

Noticed some small inconsistencies between `Time`, `Timer` and `Stopwatch` while refactoring. 

## What went wrong

This may not be totally comprehensive.

| `Time` | `Timer` | `Stopwatch` |
|-|-|-|
[elapsed_seconds](https://docs.rs/bevy/latest/bevy/prelude/struct.Time.html#method.elapsed_seconds) | [elapsed_secs](https://docs.rs/bevy/latest/bevy/prelude/struct.Timer.html#method.elapsed_secs) | [elapsed_secs](https://docs.rs/bevy/latest/bevy/time/struct.Stopwatch.html#method.elapsed_secs)
| [elapsed_seconds_f64](https://docs.rs/bevy/latest/bevy/prelude/struct.Time.html#method.elapsed_seconds_f64) | DNE  | [elapsed_secs_f64](https://docs.rs/bevy/latest/bevy/time/struct.Stopwatch.html#method.elapsed_secs_f64) | 
[is_paused](https://docs.rs/bevy/latest/bevy/prelude/struct.Time.html#method.is_paused) | [paused](https://docs.rs/bevy/latest/bevy/prelude/struct.Timer.html#method.paused) | [paused](https://docs.rs/bevy/latest/bevy/time/struct.Stopwatch.html#method.paused)

At a glance, it seems like we should
- Add `_f64` stuff to `Timer`
- Rename `*_seconds` to `*_secs` or vice versa
- Rename `paused` to `is_paused` or vice versa

## Additional information

When deciding on renames there may be other methods to look at (animation, audio, whatever) with similarly-named methods. e.g. [`AudioSink::is_paused`](https://docs.rs/bevy/latest/bevy/audio/trait.AudioSinkPlayback.html#tymethod.is_paused), [`Duration::from_secs`](https://docs.rs/bevy/latest/bevy/utils/struct.Duration.html#method.from_secs).

### PR DESCRIPTION
**Pull request by andristarr**
Fixes #15834

## Migration Guide

The APIs of `Time`, `Timer` and `Stopwatch` have been cleaned up for consistency with each other and the standard library's `Duration` type. The following methods have been renamed:

- `Stowatch::paused` -> `Stopwatch::is_paused`
- `Time::elapsed_seconds` -> `Time::elasped_secs` (including `_f64` and `_wrapped` variants)

### CODE REVIEW

**Review by alice-i-cecile — APPROVED** [review]

**Comment by alice-i-cecile** [comment]
Oh sweet, I got the nag bot working again :D

**Comment by alice-i-cecile** [comment]
I'll write the migration guide now.

File: crates/bevy_time/src/time.rs [hunk 1]
```diff
@@ -281,14 +281,14 @@ impl<T: Default> Time<T> {
     /// seconds.
     #[inline]
     pub fn delta_seconds(&self) -> f32 {
```
Comment (alice-i-cecile) [inline crates/bevy_time/src/time.rs:283]: We should rename these too actually.

**Review by alice-i-cecile — CHANGES_REQUESTED** [review]

File: crates/bevy_time/src/time.rs (hunk 1 above)
Comment (andristarr) [reply crates/bevy_time/src/time.rs]: This probably wont be too fun to review :D

**Review by ickshonpe — APPROVED** [review]
The `from_seconds` methods still need to be changed, or are you leaving the `from_` methods with the full names?

Looks good otherwise. Ran a few of the examples that use timers and nothings broken. I like `seconds` more than `secs`, but don't care that much and the uniform api is worth it.

**Review by alice-i-cecile — APPROVED** [review]

**Pull request merged**